src/module-elasticsuite-indices/Model/ResourceModel/Index/Collection.php

```diff
@@ -59,7 +59,7 @@ public function getItemsByColumnValue($column, $value)
 
         $res = [];
         foreach ($this as $item) {
-            if (strpos($item->getData($column), $value) !== false) {
+            if (str_contains($item->getData($column), $value)) {
```

Comment:
What was the problem with `strpos` ? 

`str_contains` exists only with PHP > 8.0.0 which would break BC with PHP 7.4

Reply:
This was changed due to the error in the exception.log: `Deprecated Functionality: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated in ../smile/elasticsuite/src/module-elasticsuite-indices/Model/ResourceModel/Index/Collection.php on line 62` and also according to the advice of PHPStorm `strpos()' call can be converted to 'str_contains()'`. But I forgot that `str_contains` exists only with PHP > 8.x

Reply:
Fixed.